src/components/NavigationBar.js

```diff
@@ -25,46 +30,48 @@ export default function NavigationBar() {
 
         <SideBar>
         <Link to="/" data-tip="Home" data-for="home-tooltip">
-          <SideBarItem icon={<FontAwesomeIcon icon={faHome}/>} text="Home" active alert />
+          <SideBarItem icon={<FontAwesomeIcon icon={faHome}/>} text="Home" />
         </Link>
         <Link to="/profile" data-tip="Profile" data-for="profile-tooltip">
           <SideBarItem icon={<FontAwesomeIcon icon={faUser}/>} text="Profile"
            />
         </Link>
         <Link to="/notifications" data-tip="Notifications" data-for="notifications-tooltip">
-          <SideBarItem icon={<FontAwesomeIcon icon={faBell}/>} text="Notifications" alert />
+          <SideBarItem icon={<FontAwesomeIcon icon={faBell}/>} text="Notifications" />
         </Link>
         <Link to="/create-post" data-tip="Create a Post" data-for="create-post-tooltip">
           <SideBarItem icon={<FontAwesomeIcon icon={faPlus}/>} text="Create Post" />
         </Link>
           <SideBarItem icon={<FontAwesomeIcon icon={faBookBookmark} />} text="Book Lists" />
           <SideBarItem icon={<FontAwesomeIcon icon={faPersonCircleQuestion} />} text="Reader Q&A" />
         <Link to="/settings" data-tip="Settings" data-for="settings-tooltip">
-          <SideBarItem icon={<FontAwesomeIcon icon={faGear}/>} text="Settings" alert />
+          <SideBarItem icon={<FontAwesomeIcon icon={faGear}/>} text="Settings" />
           </Link>
 
         </SideBar>
         </div>
 
-        <div className="ml-[5.2rem]">
+        <div className="ml-10">
```

Comment:
I put 5.2 rem because that's the space occupied by sideBar. Will 10 be the same as 5.2 rem?

Reply:
ml-20 would be the same, i had just made it a little smaller to match mutahhara's figma model

Reply:
Okay.